### PR TITLE
4 2 update

### PR DESCRIPTION
**Pull request by FlySkyHigh777**
Limited Teleport Update
Treewalk Update
Rune of Recall Updated
Rift Updated
Rift Self+Group Updated
Rift Lock updated
Updated Calls for: Slay, Critical Strike, Vital Blow, Proficiency, Backstab, Poison Shot, Ballista, Murder
Vital Blow Updated
Ballista Updated

### CODE REVIEW

File: rules-topics/999_changelog.md [hunk 1]
```diff
@@ -23,14 +23,17 @@
 ### Effects
 * BLEED removed.
 * LIFE updated (no longer provides immunity).
+* UNCONSCIOUS now lasts 1 minute instead of 5.
```
Comment (sd0yle) [inline rules-topics/999_changelog.md:26]: Is this reflected in life and death doc?

File: rules-topics/999_changelog.md [hunk 2]
```diff
@@ -23,14 +23,17 @@
...
 
 ### Stamina
 * Disarm updated.
-* Slay description and incant updated.
+* Slay pdated.
```
Comment (sd0yle) [inline rules-topics/999_changelog.md]: typo on "updated" :p

**Review by sd0yle — CHANGES_REQUESTED** [review]

File: rules-topics/999_changelog.md (hunk 1 above)
Comment (FlySkyHigh777) [reply rules-topics/999_changelog.md:26]: yes

File: rules-topics/999_changelog.md (hunk 2 above)
Comment (FlySkyHigh777) [reply rules-topics/999_changelog.md]: Fixed

**Review by sd0yle — APPROVED** [review]

**Pull request merged**